package.json

```diff
@@ -50,7 +50,7 @@
         "sonarqube-scanner": "^2.8.1"
     },
     "peerDependencies": {
-        "cypress": "^9.4.1"
+        "cypress": "^9.4.1 || ^10"
```

Comment:
Cypress 11 is also out. Is it possible to support it via this PR or should I create another PR?

Reply:
Added support for Cypress 11. I did some testing using and it seems to work fine.

Reply:
Appreciate your quick reaction, Thanks :)